libs/utils/analysis/tasks_analysis.py

```diff
@@ -384,7 +419,7 @@ def plotBigTasks(self, max_tasks=10, min_samples=100,
             return
 
         # Get the list of events for all big frequent tasks
-        df = self._dfg_trace_event('sched_load_avg_task')
+        df = self._dfg_task_lt_events()
         big_frequent_tasks_events = df[df.pid.isin(big_frequent_task_pids)]
```

Comment:
Still missing check for `df not None`?
Like the one you have at line 581

Reply:
Yup actually it's needed above for big_frequenct_tasks_df